.github/workflows/trivy-scan.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Set Up Ballerina
         uses: ballerina-platform/setup-ballerina@v1.1.0
         with:
-          version: latest
+          version: 2201.7.0
```

Comment:
Is there a reason for this change?

Reply:
Yes. As we will be releasing as a patch version 7.5.2, the ballerina lang version is hard coded to 2201.7.0.